src/components/inputs/DatePickerComponent.tsx

```diff
@@ -56,7 +55,7 @@ export const DatePickerComponent = ({ getDate, dateValue, disabled, isButton = f
             startIcon={<CalenderIcon2 />}
             buttonContent={
               <Typography variant="bodySm" lineHeight="20px" sx={{ color: (theme) => theme.color.gray[600] }}>
-                {value ? formatDate(value) : 'Due Date'}
+                {value ? formatDate(value) : 'Empty'}
```

Comment:
also, I think this should display Due Date in case isButton prop is true. else it can show empty.

Reply:
Thanks for pointing out! @arpandhakal